packages/cli-repl/src/cli-repl.ts

```diff
@@ -197,6 +239,28 @@ class CliRepl {
     });
   }
 
+  /**
+  * sets CliRepl.enableTelemetry based on a bool, and writes the selection to
+  * config file.
+  *
+  * @param {boolean} enabled or disabled status
+  *
+  * @returns {string} Status of telemetry logging: disabled/enabled
+  */
+  toggleTelemetry(enabled: boolean): string {
+    this.enableTelemetry = enabled;
+
+    const configPath = path.join(this.mongoshDir, 'config');
+    const config = { userId: this.userId, enableTelemetry: this.enableTelemetry };
+    fs.writeFileSync(configPath, JSON.stringify(config));
+
```

Comment:
These line are also in `generateOrReadTelemetryConfig` would it be an idea to put them together in a function? Dunno writeConfig, readConfig :/

Reply:
Good catch! I didn't even notice 😅 Will write a lil abstracted function